app/templates/confirmation.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'questionnaire.html' %}
 
-{% block page_title %}Confirmation{% endblock %}
+{% block page_title %}{{page_title}}{% endblock %}
```

Comment:
I think the idea behind having a block for the page title was that it could be set in the templates as we had no proper system of deriving the page title. Now we do, this can just be set in the `_base` template as the default value of that block (so it can be overridden at template level if needs-be)